test-configs.yaml

```diff
@@ -1016,6 +1016,23 @@ device_types:
     params: &uefi_i386
       bios_url: 'http://storage.kernelci.org/images/uefi/edk2-stable202005/i386/OVMF.fd'
 
+  qemu_riscv64:
+    base_name: qemu
+    mach: qemu
+    arch: riscv
+    boot_method: qemu
+    context:
+      arch: riscv64
+      cpu: 'rv64'
+      guestfs_interface: 'virtio'
+      machine: 'virt'
+      no_kvm: True
+      memory: 1024
```

Comment:
These things are usually configured in each lab.  Why is it different for RISC-V?

Reply:
qemu RISCV, does not work at all with kvm enabled ("qemu-system-riscv64: invalid accelerator kvm")

For the flags I believed it was added directly to "job/tags" but seems not in fact.
So I need to find another way.

Reply:
What about the memory size though?

Reply:
I didnt knew that memory was per lab. that's dangerous for reproductability

Reply:
It's set in the device dictionary in LAVA, so each QEMU instance can have a different amount.  And who knows what non-LAVA labs are doing. But then, every physical board may have a different PSU, a different firmware, a different network cable... We can't ensure everything is the same in all labs, but we can detect differences if a test fails in one lab but passes in another one.